templates/navbar-search.php

```diff
@@ -3,69 +3,70 @@
+						<div class="travel-results-navbar-subheading h6 caps line-height-4">
+							<?php echo ! empty( $_GET['s'] ) ? esc_attr( $_GET['s'] ) . '•' : ''; ?>
+							<?php echo ! empty( $_GET['start'] ) ? sprintf( 'From %s', esc_attr( $_GET['start'] ) ) : ''; ?>
+							<?php echo ! empty( $_GET['end'] ) ? sprintf( 'Until %s', esc_attr( $_GET['end'] ) ) : ''; ?>
+						</div>
+					</div>
+				</div>
 				<div class="ml3 flex items-center xs-hide sm-hide">
 
 					<div class="flex items-center">
 						<label class="travel-input-icon relative">
-							<input class="travel-input travel-input-dark rounded" type="text" name="query" placeholder="Location" on="
-					change:
-					AMP.setState({fields_query: event.value}),
-					AMP.setState({query_query: event.value})" value="<?php echo esc_attr( $_GET['s'] ); ?>">
+							<input class="travel-input travel-input-dark rounded" type="text" name="query" placeholder="<?php esc_html_e( 'Search' ); ?>" on='
+								input-throttled:AMP.setState({
+									fields_query: event.value,
+									fields_query_live: event.value,
+									fields_query_edited: query_query != event.value
+								});' value="<?php echo esc_attr( $_GET['s'] ); ?>">
 						</label>
 						<label class="travel-date-input travel-input-dark travel-date-input-touched bold relative rounded ml2">
-							<input class="block relative p0 z1" type="date" placeholder="yyyy-mm-dd" pattern="[0-9]{4}-[0-9]{2}-[0-9]{2}" title="yyyy-mm-dd" name="departure"
+							<input class="block relative p0 z1" type="date" placeholder="yyyy-mm-dd" on='
+								change:AMP.setState({
+									fields_start: event.value
+								})' pattern="[0-9]{4}-[0-9]{2}-[0-9]{2}" title="yyyy-mm-dd" name="departure"
 								value="<?php echo isset( $_GET['start'] ) ? esc_attr( $_GET['start'] ) : ''; ?>">
 
 							<div class="travel-date-input-label">
 								Departure
 							</div>
 						</label>
 						<label class="travel-date-input travel-input-dark travel-date-input-touched bold relative rounded ml2">
-							<input class="block relative p0 z1" type="date" placeholder="yyyy-mm-dd" pattern="[0-9]{4}-[0-9]{2}-[0-9]{2}" title="yyyy-mm-dd" name="return"
+							<input class="block relative p0 z1" type="date" placeholder="yyyy-mm-dd" pattern="[0-9]{4}-[0-9]{2}-[0-9]{2}" on='
+								change:AMP.setState({
+									fields_end: event.value
+								})' title="yyyy-mm-dd" name="return"
 								value="<?php echo isset( $_GET['end'] ) ? esc_attr( $_GET['end'] ) : ''; ?>">
 
 							<div class="travel-date-input-label">
-								<?php echo esc_html( 'Return', 'travel' ); ?>
+								<?php esc_html_e( 'Return', 'travel' ); ?>
 							</div>
 						</label>
+						<a href="<?php echo esc_url( site_url() ); ?>?s=<?php echo esc_attr( $_GET['s'] ); ?>" [href]="'<?php echo esc_url( site_url() ); ?>?s=' + fields_query_live + '&start=' + (fields_start || '') + '&end=' + (fields_end || '')" class="search-travel-btn travel-input travel-input-dark rounded ampstart-btn white bold">
```

Comment:
It looks like entering date value(s) removes the text from the query. For example, in the screenshot below, the query doesn't include the text:
```
https://local.them/?s=&start=2018-04-02&end=2018-04-27
```

<img width="883" alt="date-values-query" src="https://user-images.githubusercontent.com/4063887/39017800-dabf2074-43e9-11e8-9371-b7e7b22ead3b.png">

Reply:
Thanks for catching that!